felix/proto/felixbackend.proto

```diff
@@ -405,6 +405,7 @@ message WorkloadEndpoint {
   map<string, string> annotations = 11;
   QoSControls qos_controls = 12;
   LocalBGPPeer local_bgp_peer = 13;
+  map<string, string> labels = 14;
```

Comment:
I'd double check with @fasaxc if adding another non-empty possibly large map is OK just to accommodate one specific usecase. Obviously, having the label might prove handy in other places.

Reply:
That would be OK because we only do it for local workloads (so we're limited to a few hundred label maps).  

That said, I usually prefer to do the decision making in the calc graph so I might have added a dedicated field to indicate the workload type.

We do already pass the annotations through (used by VPP), so I suppose this would be consistent with that.